sekizai/templatetags/sekizai_tags.py

```diff
@@ -141,9 +157,14 @@ def render_tag(self, context, name, strip, preprocessor, nodelist):
         rendered_contents = nodelist.render(context)
         if strip:
             rendered_contents = rendered_contents.strip()
-        if preprocessor:
+        if preprocessor == 'none':
+            pass
+        elif preprocessor:
             func = import_processor(preprocessor)
             rendered_contents = func(context, rendered_contents, name)
+        elif name in _mapped_preprocessors:
+            rendered_contents = \
```

Comment:
nitpick, but I really dislike using backslash as line-continuation.

Could you change this to:

```python
rendered_contents = _mapped_preprocessors[name](
    context,
    rendered_contents,
    name
)
```